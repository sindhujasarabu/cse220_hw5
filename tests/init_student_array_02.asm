.data
_num_students: .word 1
_id_list: .word 123456
_credits_list: .word 78
_names: .ascii "John Doe\0"

.align 2
_records: .space 8  # Must be 8 * num_students

.include "p3_tester.asm"
