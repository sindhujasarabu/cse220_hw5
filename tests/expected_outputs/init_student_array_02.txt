123456 78 John Doe
